clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -43,6 +47,43 @@ public class AzureComputeClient extends AzureBaseClient {
     super(subscriptionId, azureProfile, credentials)
   }
 
+
+
+  /**
+   * Return list of available Managed VM images
+   * @param resourceGroup - filter by resource group
+   * @param region - filter by region
+   * @return List of AzureManagedVMImage
+   */
+  List<AzureManagedVMImage> getAllVMCustomImages(String resourceGroup, String region) {
+
+    def result = [] as List<AzureManagedVMImage>
+    try {
+      List<VirtualMachineCustomImage> virtualMachineCustomImages = executeOp({
+        azure.virtualMachineCustomImages()
+          .listByResourceGroup(resourceGroup)
+          .asList()
+          .stream()
+          .filter({ vm -> vm.regionName().equals(region) })
+          .collect(Collectors.toList())
+      })
+
+      virtualMachineCustomImages.each {vm ->
+        result += new AzureManagedVMImage(
+          name: vm.name(),
+          resourceGroup: vm.resourceGroupName(),
+          region: vm.regionName(),
+          osType: vm.osDiskImage().osType().name())
+      }
+
+    } catch (Exception e) {
+      log.error("getAllVMCustomImages -> Unexpected exception ", e)
```

Comment:
I just want to make sure that you are ignoring that exception on purpose, not because you forgot to handle it in a different way. :)